ledger-core/v2/virtual/integration/pending_counters_test.go

```diff
@@ -0,0 +1,266 @@
+// Copyright 2020 Insolar Network Ltd.
```

Comment:
this test is not integration, it does not require to have running virtual, so it should be placed in execute folder

Reply:
Fix it